src/Extensions/Abstractions/EventSourcesEventIds.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>Event ids enum</summary>
+	public enum EventSourcesEventIds : ushort
```

Comment:
>ushort [](start = 36, length = 6)

`ushort` isn't part of the CLS. That's not necessarily a problem, but it does make use from, say, VB.NET more difficult

Reply:
Removed exact type completely, not sure why it was here, thanks